frontend/src/scenes/instance/SystemStatus/kafkaInspectorLogic.ts

```diff
@@ -0,0 +1,37 @@
+import { kea } from 'kea'
+import api from 'lib/api'
+
+import type { kafkaInspectorLogicType } from './kafkaInspectorLogicType'
+export interface KafkaMessage {
+    topic: string
+    partition: number
+    offset: number
+    timestamp: number
+    key: string
+    value: Record<string, any> | string
+}
+
+export const kafkaInspectorLogic = kea<kafkaInspectorLogicType<KafkaMessage>>({
+    path: ['scenes', 'instance', 'SystemStatus', 'kafkaInspectorLogic'],
+    actions: () => ({
+        fetchKafkaMessage: (topic: string, partition: number, offset: number) => ({ topic, partition, offset }),
+    }),
+    loaders: () => ({
+        kafkaMessage: [
+            null as KafkaMessage | null,
+            {
+                fetchKafkaMessage: async ({ topic, partition, offset }) => {
+                    return await api.create('api/kafka_inspector/fetch_message', { topic, partition, offset })
+                },
+            },
+        ],
+    }),
+    forms: ({ actions }) => ({
+        fetchKafkaMessage: {
+            defaults: { topic: 'clickhouse_events_json', partition: 0, offset: 0 },
```

Comment:
could this be empty instead? E.g. when we no longer have that topic in the future this would error.

Reply:
probably yeah